packages/terra-table/src/TableRows.jsx

```diff
@@ -60,12 +72,13 @@ const TableRows = ({
   children,
   onClick,
   onKeyDown,
+  selectRowHelpTextId,
   ...customProps
 }) => {
   const numberOfCols = getNumberOfColumns(children);
-  const cloneChildren = cloneChildItems(children, onClick, onKeyDown, numberOfCols);
+  const cloneChildren = cloneChildItems(children, onClick, onKeyDown, numberOfCols, selectRowHelpTextId);
   return (
-    <tbody {...customProps}>
+    <tbody {...customProps} rol="rowgroup">
```

Comment:
nit: 'rol' : 'role'

Reply:
Looks like this should be `role`

Reply:
Removed altogether. https://github.com/cerner/terra-core/pull/2331/commits/a7015c82ddf086876ec90d58d94a8a8aaeb9581d